x/profile/internal/simulation/operations.go

```diff
@@ -15,7 +15,7 @@ const (
 	OpWeightMsgSaveProfile   = "op_weight_msg_save_profile"
 	OpWeightMsgDeleteProfile = "op_weight_msg_delete_profile"
 
-	DefaultGasValue = 200000
+	DefaultGasValue = 2000000
```

Comment:
We should probably make this `200000` instead as it should work properly and this is the default value that clients will use as well

```suggestion
	DefaultGasValue = 200000
```